lib/Model/Verification.cpp

```diff
@@ -674,19 +539,24 @@ static RecursiveCoroutine<bool> verifyImpl(VerifyHelper &VH,
   rc_return true;
 }
 
-RecursiveCoroutine<bool> Argument::verify(VerifyHelper &VH) const {
-  auto Guard = VH.suspendTracking(*this);
-  rc_return VH.maybeFail(CustomName().verify(VH)
-                         and rc_recur Type().verify(VH));
-}
-
 static RecursiveCoroutine<bool> verifyImpl(VerifyHelper &VH,
-                                           const CABIFunctionDefinition T) {
+                                           const CABIFunctionDefinition &T) {
   if (not T.CustomName().verify(VH)
-      or T.Kind() != TypeDefinitionKind::CABIFunctionDefinition
-      or not rc_recur T.ReturnType().verify(VH))
+      or T.Kind() != TypeDefinitionKind::CABIFunctionDefinition)
```

Comment:
Is this even possible?

Reply:
Only if something is _really_ broken, I guess.

Reply:
I'd drop it